mlflow/store/db/utils.py

```diff
@@ -17,6 +19,7 @@
     MLFLOW_SQLALCHEMYSTORE_POOL_RECYCLE,
     MLFLOW_SQLALCHEMYSTORE_MAX_OVERFLOW,
     MLFLOW_SQLALCHEMYSTORE_ECHO,
+    MLFLOW_SQLALCHEMYSTORE_DISABLE_POOLING,
```

Comment:
There are multiple Pool types sqlalchemy supports: https://docs-sqlalchemy.readthedocs.io/ko/latest/core/pooling.html#sqlalchemy.pool.Pool Consider change the flag to a string to enable future extension to other pool types.